protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -285,11 +285,17 @@ service TradingDataService {
     // Get a single proposal details
     rpc GetGovernanceData(GetGovernanceDataRequest) returns (GetGovernanceDataResponse);
 
+    // Get a single proposal
+    rpc GetProposal(GetProposalRequest) returns (GetProposalResponse);
+
     // Governance list
     //
     // List proposals using a cursor based pagination model
     rpc ListGovernanceData(ListGovernanceDataRequest) returns (ListGovernanceDataResponse);
 
+    // List proposals using a cursor based pagination model
```

Comment:
What's the difference between this and ListGovernanceData(ListGovernanceDataRequest)? 

The description is the same for both, does one need an update?